bucket/cmder.json

```
@@ -1,18 +1,22 @@
 {
-    "homepage": "http://cmder.net",
-    "description": "Portable console emulator for Windows",
+    "homepage": "https://cmder.net",
+    "description": "Portable console emulator for Windows.",
     "version": "1.3.11",
     "license": "MIT",
-    "persist": "config",
     "url": "https://github.com/cmderdev/cmder/releases/download/v1.3.11/cmder_mini.zip",
     "hash": "23c938b2495ef620de816747dfb2c0d9ad7884aec1d9923dbb7a94e6c91df849",
+    "pre_install": "if (!(Test-Path \"$dir\\vendor\\conemu-maximus5\\ConEmu.xml\")) {Copy-Item \"$dir\\vendor\\ConEmu.xml.default\" \"$dir\\vendor\\conemu-maximus5\\ConEmu.xml\"}",
```

If you use `$dir` it will be copied all the time and then renamed to `.original` and then copied from persist_dir. nonsense to check from dir instead of persist

```suggestion
    "pre_install": [
        "if (!(Test-Path \"$persist_dir\\vendor\\conemu-maximus5\\ConEmu.xml\")) {\",
        "    Copy-Item \"$dir\\vendor\\ConEmu.xml.default\" \"$dir\\vendor\\conemu-maximus5\\ConEmu.xml\"",
        "}",
    ],
```